### DISCUSSION TITLE
cache-task/

### DISCUSSION THREAD

**Discussion by giscus** [noah0316/noah0316.github.io#26] [General]
# cache-task/

들어가며

안녕하세요 Noah입니다.

SOPT라는 동아리에서 대학생들이 모여 만들게 된 앱이 공개된지 얼마되지  않았음에도 불구하고 감사하게도 많은 회원 수를 유치하게 되었습니다.

서비스 소개 페이지 : https://havit.app/

또 그만큼 돈으로 살 수 없는 소중한 피드백과, 사용자의 사용경험을 들어볼 수 있었는데요,

오늘은 사용자의 사용 경험을 높이기 위해 했던 경험 한 가지를 공유하고자

https://noah-ios.dev/cache-task/

**Comment by lsj8706** [comment]
와!!! 정말 도움 많이 됐습니다!!!
저도 캐시 고수 되고 싶네요~!!!  👍
